Brisanet/ion
# 421 create indicator

## Description

Created the Indicator Component, with three behaviors when click on button.

## How to Test
Check:
1. Whether the title changes when you pass the value to the title property;
2. The tooltip icon will appear and show the value passed to the tooltipText property
3. If displaying the value passed in the value property
4. Whether to display the secondValue passed in the secondValue property
5. ionClick will fire when you have IonIndicatorButtonType.Emitter in the buttonConfig type
6. modalEvent will fire when you have IonIndicatorButtonType.Modal in the buttonConfig type and the primary button of the modal is clicked.
7. All cases in buttonConfig, which follow IonIndicatorButtonConfiguration. You will get examples when hovering over the attributes.


## Screenshots
![image](https://user-images.githubusercontent.com/49983566/217067019-0d3bdb29-011a-4fa8-a7ba-e7c750bbf739.png)


- [ x ] I have verified that this change complies with our code and contribution policies.
- [ x ] I have verified that this change does not cause regressions and does not affect other parts of the code.
